homework-submission/week1-react/src/store/todoApp.js

```diff
@@ -0,0 +1,139 @@
+import {observable, action, runInAction} from 'mobx';
+// import todos from '../component/todos.json';
+
+
+// mobx.configure({enforceActions: true});
```

Comment:
This is the issue, if the problem as you said `MobX strict mode should be turned on!`, But this have to be `'observed'` instead of `true`